node-notes-cli/app.js

```diff
@@ -0,0 +1,38 @@
+const dataModel = require('./data.json');
+const fs = require('fs');
+
+for (const key in dataModel.notes) {
+  if (process.argv[2] === 'read') {
+    console.log(`${key}: ${dataModel.notes[key]}`);
+  }
+}
+
+if (process.argv[2] === 'create') {
+  const key = dataModel.nextId;
+  dataModel.notes[key] = process.argv[3];
+  dataModel.nextId++;
+  const dataObject = JSON.stringify(dataModel, null, 2);
+  fs.writeFile('data.json', dataObject, 'utf8', err => {
+    if (err) throw err;
+  });
+}
+
+// for (const key in dataModel.notes) {
```

Comment:
Please do not turn in commented out code 😱

Reply:
I'm sorry 😓